<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-R8U126V" name="PassageWarning" tests="4" failures="1" timestamp="10 Feb 2020 13:26:03 GMT" time="3.895" errors="0">
  <testcase name="createPassageWarningAuditVerify" time="0.897" classname="com.Tests.PassageWarning">
    <failure type="java.lang.AssertionError" message="expected [200] but found [400]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [400]
	at com.Utilities.Assertion.assertEquals(Assertion.java:31)
	at com.InitialSetup.BaseClass.verifyResponseCode(BaseClass.java:338)
	at com.Tests.PassageWarning.createPassageWarningAuditVerify(PassageWarning.java:45)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- createPassageWarningAuditVerify -->
  <testcase name="createPassageWarningAuditBlankDataVerify" time="0.818" classname="com.Tests.PassageWarning"/>
  <testcase name="getPassageWarningAuditVerify" time="0.842" classname="com.Tests.PassageWarning"/>
  <testcase name="getPassageWarningVerify" time="1.018" classname="com.Tests.PassageWarning"/>
</testsuite> <!-- PassageWarning -->
